hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -82,7 +73,7 @@ public void run(HookContext hookContext) {
 
 	    } catch (Throwable e) {
 		    LOGGER.error("Error in executing LineageHook", e);
-			e.printStackTrace(System.out);
+			e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1148)